### PR TITLE
Treat descriptions as formatted using markdown

### PR DESCRIPTION
**Pull request by rbruggem**
This change ensures markdown descriptions in openapi files are correctly formatted in vuepress.

Compare https://github.com/lune-climate/lune-docs/pull/50 `/api-reference/simpleshippingmethod.html` to this branch's preview.

Includes https://github.com/lune-climate/lune-docs/pull/50.

### CODE REVIEW

**Comment by rbruggem** [comment]
Rebased https://github.com/lune-climate/lune-docs/pull/50

File: docs/src/.vuepress/components/Md.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  props: {
+    markdown: {
+      type: String,
+    },
+  },
+  computed: {
+    renderedMarkdown() {
+        if (!this.markdown) {
+            return undefined
+        }
```
Comment (jstasiak) [inline docs/src/.vuepress/components/Md.vue:18]: Is this conditional block actually needed? It seems like the only false-ish value allowed is an empty string and that should render as Markdown just fine

**Review by jstasiak — APPROVED** [review]
LGTM

File: docs/src/.vuepress/components/Md.vue (hunk 1 above)
Comment (rbruggem) [reply docs/src/.vuepress/components/Md.vue:18]: It is.
All members of `this.` are undefined on the first render.

**Pull request merged**